<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchChanges</class>
 <widget class="QDialog" name="SearchChanges">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search for recent changes</string>
  </property>
  <widget class="QLabel" name="labelSelectRepository">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select repository:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxRepos">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Fusion for Office V1R8</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>349</x>
     <y>21</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Define a period from:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>799</x>
     <y>16</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>728</x>
     <y>16</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowserSVNChanges">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>851</width>
     <height>571</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditFrom">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>19</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
   <property name="date">
    <date>
     <year>2019</year>
     <month>5</month>
     <day>6</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>542</x>
     <y>20</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>to:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEditTo">
   <property name="geometry">
    <rect>
     <x>562</x>
     <y>19</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy/MM/dd</string>
   </property>
   <property name="date">
    <date>
     <year>2019</year>
     <month>5</month>
     <day>22</day>
    </date>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
